### PR TITLE
Formatted attributes

### LINKED ISSUE #38: formatted_attributes for Export
``` ruby
def formatted_attributes
 # same as attributes, but with attributes calling `format_cell`
end

def formatted_attribute(column_name)
   self.class.format_cell(.......)
end

# change regular attributes to not call `format_cell`
```

### PR DESCRIPTION
**Pull request by joel**
closes #38

### CODE REVIEW

File: lib/csv_row_model/model/columns.rb [hunk 1]
```diff
@@ -17,10 +17,18 @@ def headers
...
 
       protected
+
       def attributes_from_column_names(column_names)
+        hash_convert(column_names) { |column_name| public_send(column_name) }
+      end
+
+      private
+
+      def hash_convert(column_names)
```
Comment (s12chung) [inline lib/csv_row_model/model/columns.rb]:
``` ruby
def map_array_to_block(array, &block)
# change yield(column) to block.call(column_name) --- it's not different, just to make it super clear
end
```

i like the method :)

File: lib/csv_row_model/export/attributes.rb [hunk 2]
```diff
@@ -0,0 +1,52 @@
+module CsvRowModel
+  module Export
+    module Attributes
+      extend ActiveSupport::Concern
+
+      included do
+        self.column_names.each { |*args| define_attribute_method(*args) }
+      end
+
+      def formatted_attributes
```
Comment (s12chung) [inline lib/csv_row_model/export/attributes.rb:11]: please change:

``` ruby
# in export
def to_row
formatted_attributes.values
end
```

**Comment by s12chung** [comment]
overall looks great :white_check_mark: 

just one thing:
you moved methods from `export.rb` --> `attributes.rb`
can you do the same for `export_spec.rb` --> `attributes_spec.rb`?

File: lib/csv_row_model/model/columns.rb (hunk 1 above)
Comment (joel) [inline lib/csv_row_model/model/columns.rb]: `map_array_to_hash`?

Comment (s12chung) [inline lib/csv_row_model/model/columns.rb]: yours is a noun. it's better than my verb :D `array_to_block_hash`?

**Comment by s12chung** [comment]
going to make a branch for #37

**Pull request merged**